src/NuGet.Clients/NuGet.PackageManagement.UI/UserAction.cs

```diff
@@ -21,12 +21,28 @@ private UserAction(NuGetProjectActionType action, string packageId, NuGetVersion
             Version = packageVersion;
         }
 
+        private UserAction(NuGetProjectActionType action, string packageId, NuGetVersion packageVersion, VersionRange versionRange)
+        {
+            Action = action;
+
+            if (string.IsNullOrEmpty(packageId))
+            {
+                throw new ArgumentNullException(nameof(packageId));
+            }
+
+            PackageId = packageId;
+            Version = packageVersion;
```

Comment:
This entire block of code can be removed if we can call `this (action, packageId, packageVersion)`.